cloudbuild/nightly/scripts/python-scripts/create_unbounded_sink_table.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+from collections.abc import Sequence
+
+from absl import app
+from google.cloud import bigquery
+from absl import logging
+from google.cloud.bigquery import DatasetReference
+
+
+def create_destination_table(project_name, dataset_name, destination_table_name):
+    # Construct a BigQuery client object.
+    client = bigquery.Client(project=project_name)
+
+    table_id = f"{project_name}.{dataset_name}.{destination_table_name}"
+    # Define the BigQuery schema
+    table_schema = [
+        bigquery.SchemaField("unique_key", "STRING", mode="REQUIRED"),
+        bigquery.SchemaField("name", "STRING", mode="REQUIRED"),
+        bigquery.SchemaField("number", "INTEGER", mode="REQUIRED"),
+        bigquery.SchemaField("ts", "TIMESTAMP", mode="REQUIRED"),
+    ]
+    table = bigquery.Table(table_id, table_schema)
+    try:
+        table = client.create_table(table)  # Make an API request.
+        logging.info(
+            "Created table {}.{}.{}".format(table.project, table.dataset_id, table.table_id)
+        )
+    except Exception as _:
+        raise RuntimeError("Table could not be created!")
+
+
+def main(argv: Sequence[str]) -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--project_name',
+        dest='project_name',
+        help='Project Id which contains the table to be read.',
```

Comment:
`Project Id for creation of destination table.`
similar for `--dataset_name`

Reply:
Missed changing it. Thanks for pointing out! Modified it.